src/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mojaloop/finance-portal-backend-service",
-  "version": "9.1.0",
+  "version": "9.1.1",
```

Comment:
We forgot to bump the version in our PR, so it's actually more like 9.1.2 I think.